.github/workflows/main.yml

```diff
@@ -80,9 +84,22 @@ jobs:
       run: |
         echo ::set-env name=CC::gcc-8
         echo ::set-env name=CXX::g++-8
-    - name: Run the tests
+    - name: Run the tests (osx)
+      if: startsWith(matrix.os, 'macos')
+      run: |
+        ulimit -c unlimited
+        make -j2 test
+    - name: Run the tests (ubuntu)
+      if: startsWith(matrix.os, 'ubuntu')
       run: |
+        ulimit -c unlimited
+        sudo chmod -R +rwx /var/crash
```

Comment:
we can lift just this one line to an ubuntu specific thing and then have then ulimit and make command in non-branched command, right?